content/tracing/advanced_usage/debugging.md

```diff
@@ -10,6 +10,9 @@ Datadog debug settings are used to diagnose issues or audit trace data.
 To return debug level application logs, enable debug mode with the flag `-Ddatadog.slf4j.simpleLogger.defaultLogLevel=debug` when starting the JVM.
 {{% /tab %}}
 {{% tab "Python" %}}
+Debugging is disabled by default. To enable it set the environment variable
+`DATADOG_TRACE_DEBUG=true` when using `ddtrace-run`. Note that this will
+disable traces from being sent to Datadog.
```

Comment:
This doesn't disable the tracer. It will just print way more details about the traces collection. Also you must add a notice saying "Don't use this option in your high traffic services because it may increase the number of produced log lines".